### ISSUE TITLE
--application-definition-kind-filter documentation confusing

### ISSUE THREAD

**Issue by Jose-CF** [Azure/service-fabric-cli#39]
I went through this text few times:
![image](https://user-images.githubusercontent.com/24405049/30945942-418dcbfc-a42b-11e7-9f52-10fe425ea0de.png)
And found this snippet unfairly complicated to read.
I know my intellect is not very powerful. But I bet this could be a bit more clear. At least the indentation would help.

**Comment by linggengmsft** [comment]
@Jose-CF 
We added definition kind in the Application and Application Type query in v6.0 release:
In addition to the existing application package deployments, we added support for compose deployments (preview). To distinguish between different kind of deployments, the application information returned by get application list query exposes an additional “ApplicationDefinitionKind” field. Similarly, the application type information returned by the get application type list query includes “ApplicationTypeDefinitionKind" field. 
This argument as well as the similar filter "application-type-definition-kind-filter" in application type query are for filtering this new definition kind.

We will keep enhancing the helper message and fix the indent issue.
